File: 1046.py
x = input().split()
i, f = x

i = int(x[0])
f = int(x[1])

if i < f:
    t = f - i
else:
    t = (24 - i) + f
print('O JOGO DUROU {} HORA(S)'.format(t))

File: CSharp/1046.cs
using System;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
       int[] lista = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int a=lista[0];
        int b=lista[1];

        if(a<b)
        {
            a=b-a;
        }
        else if(a>b)
        {
            a= 24-a+b;
        }
        else{a=24;}

        Console.WriteLine($"O JOGO DUROU {a} HORA(S)");
    }
}